edx-olive/themes
# [SE-3702] remove support tab from LMS theme

Remove the support tab from the theme used on LMS.

**Dependencies**: None

**Screenshots**:

![Screenshot 2021-03-09 at 12 16 42](https://user-images.githubusercontent.com/19173947/110463222-eb5a0e80-80d1-11eb-8c07-009058bdbc44.png)

**Sandbox URL**: N/A

**Merge deadline**: None

**Testing instructions**:

1. Enable stage cluster v135
2. Navigate to [login](https://courses.stage.campus.gov.il/login?next=/home%2Fen%2F)
3. Verify no support tab appears

**Author notes and concerns**:

N/A

**Reviewers**
